js/main.js

```diff
@@ -14,8 +14,102 @@ function getFromTo (min, max) {
   return (scalar * Math.random()) + myMin;
 }
 window.console.log(getFromTo (1,100));
+*/
+function getRandomPositiveInteger (a, b) {
+  const lower = Math.ceil(Math.min(Math.abs(a), Math.abs(b)));
+  const upper = Math.floor(Math.max(Math.abs(a), Math.abs(b)));
+  const result = Math.random() * (upper - lower + 1) + lower;
+  return Math.floor(result);
+}
+getRandomPositiveInteger (0,25);
 
 function checkStringLength (string, length) {
   return string.length <= length;
 }
 window.console.log (checkStringLength (110, 150));
+
+// module4-task1
+
+const urls = [
```

Comment:
например это так не Льзя оставлять

```suggestion
const urls = Array.from({length: 25}, (_,ix)=>`photos/${1+ix}.jpg`);
```

Reply:
Т.е. вместо 25 строк - одна